RUFAS/simulation_engine.py

```diff
@@ -86,28 +100,36 @@ def _run_post_annual_routines(self) -> None:
         """
         Writes the annual report to the output files
         Flushes the data in the output object
-        Resets the state for the following year"""
+        Resets the state for the following year
+        """
+
         self.state.annual_mass_balance(self.time)
         self.state.annual_reset()
         self.time.advance()
 
-    def _visualize_sim_progress(self, day: int, update_interval: int = 50) -> None:
+    @staticmethod
+    def _visualize_sim_progress(day: int, update_interval: int = 50) -> None:
         """
         Shows a rotating char on console to confirm simulation is alive.
 
-        Args:
-            day (int): day of year
-            update_interval (int): the interval at which the char symbol is updated
+        Parameters
+        ----
+        day : int
+            day of year
+        update_interval : int
+            the interval at which the char symbol is updated. Default is 50.
```

Comment:
```suggestion
            Ordinal day of the year.
        update_interval : int, default=50
            The interval at which the char symbol is updated.
```
Adding units if known for `update_interval` would be good.